rector.php

```diff
@@ -1,42 +1,47 @@
 <?php
 
+use PhpParser\Node\Expr\Cast\Bool_;
+use PhpParser\Node\Expr\Cast\Double;
+use PhpParser\Node\Expr\Cast\Int_;
 use Rector\Config\RectorConfig;
 use Rector\DeadCode\Rector\ClassLike\RemoveAnnotationRector;
 use Rector\Php70\Rector\StmtsAwareInterface\IfIssetToCoalescingRector;
 use Rector\Php71\Rector\FuncCall\RemoveExtraParametersRector;
 use Rector\Php80\Rector\Switch_\ChangeSwitchToMatchRector;
-use Rector\PHPUnit\PHPUnit100\Rector\Class_\StaticDataProviderClassMethodRector;
-use Rector\PHPUnit\Set\PHPUnitSetList;
-use Rector\Set\ValueObject\LevelSetList;
+use Rector\Renaming\Rector\Cast\RenameCastRector;
+use Rector\Renaming\ValueObject\RenameCast;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths([
+return RectorConfig::configure()
+    ->withPaths([
         __DIR__ . '/examples',
         __DIR__ . '/src',
         __DIR__ . '/tests',
         __DIR__ . '/tools',
-    ]);
-
-    // Modernize code
-    $rectorConfig->sets([
-        LevelSetList::UP_TO_PHP_74,
-        PHPUnitSetList::PHPUNIT_100,
-    ]);
-
-    $rectorConfig->rule(ChangeSwitchToMatchRector::class);
-    $rectorConfig->rule(StaticDataProviderClassMethodRector::class);
-
+    ])
+    ->withPhpSets(php74: true)
```

Comment:
`php74: true` could be dropped in order to automatically upgrade the code to the PHP version defined in `composer.json`, 8.1 in this case. Doing this changes over 250 files.